### PR TITLE
Make it compile with substrate beta

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: src/chain_spec.rs [hunk 1]
```diff
@@ -19,13 +22,22 @@ pub enum Alternative {
 	LocalTestnet,
 }
 
+const MILLICENTS: u128 = 1_000_000_000;
```
Comment (gavofyork) [inline src/chain_spec.rs]: this is meant to be a minimal example of a runtime - there's no need to add this stuff

File: src/chain_spec.rs [hunk 2]
```diff
@@ -93,11 +103,42 @@ fn testnet_genesis(initial_authorities: Vec<AuthorityId>, endowed_accounts: Vec<
 			creation_fee: 0,
 			reclaim_rebate: 0,
 			balances: endowed_accounts.iter().map(|&k|(k, (1 << 60))).collect(),
-			_genesis_phantom_data: Default::default(),
 		}),
 		upgrade_key: Some(UpgradeKeyConfig {
 			key: upgrade_key,
-			_genesis_phantom_data: Default::default(),
+		}),
+		democracy: Some(DemocracyConfig {
```
Comment (gavofyork) [inline src/chain_spec.rs]: same - there's no need to bring in all these governance modules.

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -392,22 +246,43 @@ impl_runtime_apis! {
 			Executive::finalise_block()
 		}
 
-		fn inherent_extrinsics(data: InherentData) -> Vec<UncheckedExtrinsic> {
-			data.create_inherent_extrinsics()
+		fn inherent_extrinsics(data: BasicInherentData) -> Vec<<Block as BlockT>::Extrinsic> {
+			let mut inherent = Vec::new();
```
Comment (gavofyork) [inline runtime/src/lib.rs]: why is this now manually done?

Comment (bkchr) [reply runtime/src/lib.rs]: We still don't have any new way for doing this automatically.

Comment (gavofyork) [reply runtime/src/lib.rs]: didn't `data.create_inherent_extrinsics()` do it automatically?

**Pull request merged**